modules/bmaas-using-the-bmo-to-manage-resources-across-all-namespaces.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bmaas-using-the-bmo-to-manage-resources-across-all-namespaces_{context}"]
+= Using the Bare Metal Operator to manage resources across all namespaces
+
+For the Bare Metal Operator (BMO) to manage `BareMetalHost` resources across all namespaces in your {product-title} cluster, you must configure the Operator to watch all namespaces. This configuration is important to avoid mixing non-{product-title} workloads with other components in the same namespace.
+
+.Prerequisites
+
+* If you are using user-provisioned installation and the Provisioning CR does not exist, you must create it manually. For instructions, see https://docs.openshift.com/container-platform/{product-version}/installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.html#configuring-a-provisioning-resource-to-scale-user-provisioned-clusters_scaling-a-user-provisioned-cluster-with-the-bare-metal-operator[Configuring a provisioning resource to scale user-provisioned clusters]. For installer-provisioned installations, the installation program creates the Provisioning custom resource (CR) automatically.
```

Comment:
Docs.openshift.com is no longer live and links need to be prefixed with `link:`.  Please add the `link:` prefix and switch to a d.r.c link in a follow up PR.

Ideally, this link and others linking to the internal docs would be an xref in an additional resources section following the module as a trailing include, but I'm not going to block on it.